addons/account/wizard/account_move_send.py

```diff
@@ -208,7 +208,7 @@ def _get_placeholder_mail_attachments_data(self, move):
 
     @api.model
     def _get_placeholder_mail_template_dynamic_attachments_data(self, move, mail_template):
-        invoice_template = self.pdf_template_id
+        invoice_template = self and self.pdf_template_id or self._get_default_pdf_template_id()
```

Comment:
Yeah, I really hate to have an access to `self` in an `@api.model` method. 
Can't you pass it to an optional parameter instead ?

Reply:
Optional parameters don't break stable policy since the call can still happen without them by definition.